tests/quadsim/robots/test_robots.py

```diff
@@ -0,0 +1,103 @@
+"""Unit tests for the motor model class."""
+import numpy as np
+import pybullet
+import pybullet_data
+from pybullet_utils import bullet_client
+
+from quadsim.robots.a1 import A1
+from quadsim.simulator import SimulatorConf
+
+
+def setup():
```

Comment:
Feel free to commit and create an issue for this

Looks like this will create multiple pybullet instances. Can we create just one bullet instance and reuse it across tests?